packages/init/init.ts

```diff
@@ -16,12 +16,12 @@ const propTypes = require("@webpack-cli/utils/prop-types");
  *
  * @param	{Object} transformObject 	- An Object with all transformations
  * @param	{Object} config 			- Configuration to transform
- * @returns {Object} - An Object with the transformations to be run
+ * @returns {Array} - An array with the transformations to be run
  */
 
-function mapOptionsToTransform(config) {
-	return Object.keys(config.webpackOptions).filter(k => propTypes.has(k));
-}
+const mapOptionsToTransform = (config): string[] =>
```

Comment:
Why it's been changed into a `const`?

Reply:
No particular reason. We can decide on some conversion and follow same all across the code.

Reply:
Sounds good to me! Always function unless there are some scope related things (`this.`)?

Reply:
Why?

Reply:
Is it mutated? I recall not

Reply:
What?

Reply:
No the function is pure

Reply:
Then const is good :)

Reply:
Are you basically saying that pure functions should be written with `const puppy = () =>{}` and impure functions should be written with `function puppy() {}`?

Reply:
If the car is modified after declaration then using let is better.

Reply:
OK unless there's a misunderstanding, I just give up

Reply:
Hey, guys. I think there is some misunderstanding. :)
Reassigning variables and purity of functions are different concepts. Reassignment is about mutability and purity is about input (the arguments) and output (return value) of a function.

Reply:
I know, my point was why `function ()` was changed to `const pp = () => {}`. If so, let's just agree on a convention on when having one or the other. I don't care if it's `const` or `let` :no_good_man: . But that's OK, no need to go further